tst/EnergyPlus/unit/AirLoopHVACDOAS.unit.cc

```diff
@@ -4462,4 +4464,4007 @@ TEST_F(EnergyPlusFixture, AirLoopHVACDOAS_TestGetDesignDayConditions)
+
+    state->dataSurfaceGeometry->CosBldgRotAppGonly = 1.0;
+    state->dataSurfaceGeometry->SinBldgRotAppGonly = 0.0;
+    state->dataSurfaceGeometry->CosZoneRelNorth.allocate(6);
+    state->dataSurfaceGeometry->SinZoneRelNorth.allocate(6);
+    state->dataSurfaceGeometry->CosZoneRelNorth = 1.0;
+    state->dataSurfaceGeometry->SinZoneRelNorth = 0.0;
+    state->dataSurfaceGeometry->CosBldgRelNorth = 1.0;
+    state->dataSurfaceGeometry->SinBldgRelNorth = 0.0;
+    SurfaceGeometry::GetSurfaceData(*state, ErrorsFound);
+    EXPECT_FALSE(ErrorsFound);
+
+    ZoneEquipmentManager::GetZoneEquipment(*state);
+    SimAirServingZones::GetAirPathData(*state);
+    // OA inlet node
+    auto &PSZAC_OAInNode_1 = state->dataLoopNodes->Node(
+        UtilityRoutines::FindItemInList("PSZ-AC:1_OAINLET NODE", state->dataLoopNodes->NodeID, state->dataLoopNodes->NumOfNodes));
+    auto &PSZAC_OAInNode_2 = state->dataLoopNodes->Node(
+        UtilityRoutines::FindItemInList("PSZ-AC:2_OAINLET NODE", state->dataLoopNodes->NodeID, state->dataLoopNodes->NumOfNodes));
+    auto &PSZAC_OAInNode_3 = state->dataLoopNodes->Node(
+        UtilityRoutines::FindItemInList("PSZ-AC:3_OAINLET NODE", state->dataLoopNodes->NodeID, state->dataLoopNodes->NumOfNodes));
+    auto &PSZAC_OAInNode_4 = state->dataLoopNodes->Node(
+        UtilityRoutines::FindItemInList("PSZ-AC:4_OAINLET NODE", state->dataLoopNodes->NodeID, state->dataLoopNodes->NumOfNodes));
+    auto &PSZAC_OAInNode_5 = state->dataLoopNodes->Node(
+        UtilityRoutines::FindItemInList("PSZ-AC:5_OAINLET NODE", state->dataLoopNodes->NodeID, state->dataLoopNodes->NumOfNodes));
+    // set OA inlet node flow rates
+    PSZAC_OAInNode_1.MassFlowRate = 0.1;
+    PSZAC_OAInNode_2.MassFlowRate = 0.1;
+    PSZAC_OAInNode_3.MassFlowRate = 0.1;
+    PSZAC_OAInNode_4.MassFlowRate = 0.1;
+    PSZAC_OAInNode_5.MassFlowRate = 0.1;
+
+    // OA relief node
+    auto &PSZAC_OARelNode_1 = state->dataLoopNodes->Node(
+        UtilityRoutines::FindItemInList("PSZ-AC:1_OARELIEF NODE", state->dataLoopNodes->NodeID, state->dataLoopNodes->NumOfNodes));
+    auto &PSZAC_OARelNode_2 = state->dataLoopNodes->Node(
+        UtilityRoutines::FindItemInList("PSZ-AC:2_OARELIEF NODE", state->dataLoopNodes->NodeID, state->dataLoopNodes->NumOfNodes));
+    auto &PSZAC_OARelNode_3 = state->dataLoopNodes->Node(
+        UtilityRoutines::FindItemInList("PSZ-AC:3_OARELIEF NODE", state->dataLoopNodes->NodeID, state->dataLoopNodes->NumOfNodes));
+    auto &PSZAC_OARelNode_4 = state->dataLoopNodes->Node(
+        UtilityRoutines::FindItemInList("PSZ-AC:4_OARELIEF NODE", state->dataLoopNodes->NodeID, state->dataLoopNodes->NumOfNodes));
+    auto &PSZAC_OARelNode_5 = state->dataLoopNodes->Node(
+        UtilityRoutines::FindItemInList("PSZ-AC:5_OARELIEF NODE", state->dataLoopNodes->NodeID, state->dataLoopNodes->NumOfNodes));
+    // set OA relief node data
+    PSZAC_OARelNode_1.MassFlowRate = 0.1;
+    PSZAC_OARelNode_2.MassFlowRate = 0.1;
```

Comment:
@Nigusse And nice approach here, referencing the node, robust and readable! R&R

Reply:
@mjwitte Thanks. You suggested the idea. Mostly, there is better or best way of doing things.